### ISSUE TITLE
Pretrained models problem

### ISSUE THREAD

**Issue by SkeletonOne** [sxhxliang/detectron2_backbone#5]
Appreciate your great work.

In terms of your README, I can only find the pretrained efficientnet model. So where can I download other pretrained models. For example, resnet18, and use the convert_weight.py to convert it for training?

**Comment by sxhxliang** [comment]
you need to convert the pre-trained model yourself. 

If you use fpn, the pre-trained  model need to add prefix name, for example, the first convolutional layer of resnet18 is `conv1` ， you need to map the key (`conv1`) to a new key(`backbone.bottom_up.stem.conv1`) for detectron2 


`
  (backbone): FPN(
    (fpn_lateral3): Conv2d(128, 256, kernel_size=(1, 1), stride=(1, 1))
    (fpn_output3): Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (fpn_lateral4): Conv2d(256, 256, kernel_size=(1, 1), stride=(1, 1))
    (fpn_output4): Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (fpn_lateral5): Conv2d(512, 256, kernel_size=(1, 1), stride=(1, 1))
    (fpn_output5): Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
    (top_block): LastLevelP6P7(
      (p6): Conv2d(256, 256, kernel_size=(3, 3), stride=(2, 2), padding=(1, 1))
      (p7): Conv2d(256, 256, kernel_size=(3, 3), stride=(2, 2), padding=(1, 1))
    )
    (bottom_up): ResNet(
      (stem): BasicStem(
        (conv1): Conv2d(
          3, 64, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3), bias=False
          (norm): FrozenBatchNorm2d(num_features=64, eps=1e-05)
        )
      )
`

**Comment by sxhxliang** [comment]
I've simply updated the code. 



```
wget https://download.pytorch.org/models/resnet18-5c106cde.pth
python3 -m detectron2_backbone.tools.convert_weight  --model resnet18-5c106cde.pth --output  resnet18-d2.pth
```
```
your_config.yaml
MODEL:
  WEIGHTS: "resnet18-d2.pth"

```
It is easy to convert the pre-trained model now.

**Comment by SkeletonOne** [comment]
@sxhxliang Thanks for your reply, it is great to use!
